packages/samples/graph_data/test/nestedLoop.yaml

```
@@ -0,0 +1,37 @@
+        result:
+          value: []
+          update: :reducer.array
+          isResult: true
+        retriever:
+          agent: shiftAgent
+          inputs:
+            array: :people
+        query:
+          agent: copyAgent
+          inputs:
+            data: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)